### DISCUSSION TITLE
空腹の表示位置をHPなどの近くにしてほしい

### DISCUSSION THREAD

**Discussion by PPMDer** [hengband/hengband#3131] [Ideas]
お疲れ様です。
現在の空腹の表示位置は左下隅で個人的にかなり見づらいと感じています。自分はBGMやSEはなしでプレイしているので、いつの間にか衰弱状態に陥ってしまって麻痺がおこります。もちろん気を付けていれば起こることはない事故だとは思いますが、以前の表示方法ならばそういったことは起こりませんでした。

空腹状態をＨＰなどの近くに表示していただければかなり助かるのですが、どうでしょうか？
一度変更があったものにとやかく言うことになってしまい、申し訳ございません。

**Comment by tanguband** [comment]
[Fix] メインウィンドウ左のプレイヤーの状態表示 https://github.com/hengband/hengband/pull/3122

こちらで修正が入っているようなので次のバージョンで反映されると思います。

**Comment by habu1010** [comment]
位置を修正してしばらく経ちますが、現在の位置でとくに問題あるという意見は出ていなそうなのでクローズします。
